bin/live/pycbc_live_single_trigger_fits

```diff
@@ -60,6 +61,16 @@ parser.add_argument("--duration-bin-spacing",
                     help="How to set spacing for bank split "
                          "if using --duration-bin-start, --duration-bin-end "
                          "and --num-duration-bins.")
+parser.add_argument('--prune-loudest', type=int,
+                    help="Number of loudest triggers to remove from each "
```

Comment:
seems slightly convoluted - how about '[Maximum] number of loudest trigger clusters to remove from each bin' ?
Maximum because if the loudest are not all above the threshold the number removed will be less